lib/msf/ui/console/command_dispatcher/auxiliary.rb

```diff
@@ -48,9 +55,27 @@ def method_missing(meth, *args)
 
       return mod.send(meth.to_s, *args)
     end
+
+    action = meth.to_s.delete_prefix('cmd_')
+    if mod && mod.kind_of?(Msf::Module::HasActions) && mod.actions.map(&:name).map(&:downcase).include?(action)
+       do_action(action, *args)
```

Comment:
Since `do_action` calls `cmd_run` and the return value sometimes matters, we should return it here.

```suggestion
       return do_action(action, *args)
```